cli-commands/deployer/deployer.js

```diff
@@ -47,10 +51,15 @@ const run = async (deploymentFilePath, network, secret, silent, compile, runs) =
 	}
 
 	const currentRecord = records[records.length - 1];
-	console.log(`\nHere is your report:`);
-	utils.printReportTable(currentRecord.actions);
-}
+	logger.log(`\nHere is your report:`);
+
+	if (!output || output === loggerAppenderTypes.NORMAL) {
```

Comment:
Now that you've supplied default value of the output, I do not think you need `!output`